docs/guides/README.md

```diff
@@ -2,6 +2,8 @@
 
 Here you'll find some guides that will help you work with (and understand better) different topics related to Frontity. 
```

Comment:
Replace
"Here you'll find some guides that will help you work with (and understand better) different topics related to Frontity."
with
"In this section we present a selection of guides that will help you to both work with, and to better understand, a variety of topics related to Frontity."